src/libs/filter-utils.ts

```diff
@@ -58,10 +86,23 @@ export namespace FilterUtilities {
     if (personId !== undefined && person.toString() !== personId) {
       return false;
     }
-
     return true;
   }
 
+  /**
+   * Compares severa User id to targetUser id
+   * 
+   * @param severaUserId User id from Severa
+   * @param targetUserId User id to compare
+   * @returns 
```

Comment:
Remove unnecessary @returns doc and the explicit return typing. I'm also not sure that this filter method should be necessary after getWorkHoursHandler restructuring.
Even if it is needed as the logic is quite short I wonder if it wouldn't just be clearer to do it inline.